.vscode/pnpify/eslint/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "eslint",
+  "version": "6.8.0-pnpify",
+  "main": "./lib/api.js"
+}
```

Comment:
The files in `pnpify` were generated by the commands from the [docs](https://yarnpkg.com/advanced/editor-sdks#vscode).